packages/block-library/src/navigation-menu/edit.js

```diff
@@ -73,7 +74,7 @@ function NavigationMenu( {
 				return null;
 			}
 
-			return pages.map( ( { title, type, link: url, id } ) => (
+			return pages.map( ( { title, type, link: url, id, menuEventActivation } ) => (
```

Comment:
Wondering whether `menuEventActivation` would be clearer as `dropdownActivationEvent`. At the least swapping `Event` and `Activation` should happen to make the "thing" we are describing (ie: the event) is clearer (ie: `menuActivationEvent`).